### PR TITLE
Nocturnalmc

### PR DESCRIPTION
**Pull request by nocturnalmc**
- Preparation for version bump v2.1.0
- Added TODO for later work (can hotpatch before production maybe)

### CODE REVIEW

File: client/src/user/components/sekolah/UserSekolah.jsx [hunk 1]
```diff
@@ -154,16 +153,13 @@ function UserSekolah() {
         }
       )
       .then((res) => {
-        console.log(res);
         setModalBegin(false);
         setReloadState(!reloadState);
         setSubmittingBegin(false);
```
Comment (c4lyp5o) [inline client/src/user/components/sekolah/UserSekolah.jsx:158]: @nocturnalmc nk pakai try catch finally x?

**Review by c4lyp5o — COMMENTED** [review]
Comments appended

File: client/src/user/components/sekolah/UserSekolah.jsx (hunk 1 above)
Comment (nocturnalmc) [reply client/src/user/components/sekolah/UserSekolah.jsx:158]: @c4lyp5o Dalam toast.promise tak boleh dato, kalau guna try catch nnti error toast tu xleh masukkan dalam catch block. Comment atas tu aku silap tp toast.promise ni @Dvachini din dah refactor haritu

**Review by nocturnalmc — COMMENTED** [review]
Streamline guna toast.promise seperti UserFormUmumHeader punya handleSubmit

**Review by c4lyp5o — APPROVED** [review]
Ok alhamdulillah masyuk

**Pull request merged**